Sources/TuistLoader/Utils/ManifestFilesLocator.swift

```diff
@@ -183,7 +183,7 @@ public final class ManifestFilesLocator: ManifestFilesLocating {
     public func locatePackageManifest(at locatingPath: AbsolutePath) -> AbsolutePath? {
         let subPath =
             // swiftlint:disable:next force_try
-            try! RelativePath(validating: "\(Constants.tuistDirectoryName)/Package.swift")
+            try! RelativePath(validating: Constants.SwiftPackageManager.packageSwiftName)
```

Comment:
I'd require the `Package.swift` to be next to the `Tuist/` directory, which represents the whole workspace and make sure our documentation reflects that. Otherwise, users might get unexpected behaviours if our "traverse up" logic finds a `Package.swift` in the sub-directory or any of the intermediate ones which is not the one that the user expects.